src/Graphics/src/Graphics/NumericExtensions.cs

```diff
@@ -24,11 +29,15 @@ public static float Clamp(this float self, float min, float max)
 			}
 
 			return self;
+#endif
 		}
 
 		[MethodImpl(MethodImplOptions.AggressiveInlining)]
 		public static double Clamp(this double self, double min, double max)
 		{
+#if NETCOREAPP3_0_OR_GREATER
+			return double.Clamp(self, min, max);
+#else
```

Comment:
Reading the docs, I think `double/float.Clamp()` were introduced in .NET 7?

https://learn.microsoft.com/en-us/dotnet/api/system.double.clamp?view=net-7.0